### DISCUSSION TITLE
Logging scores on the training set

### DISCUSSION THREAD

**Discussion by oroszgy** [explosion/spaCy#7600] [Language Support]
Hi there, I am in the middle of training new UD models for Hungarian, and would need some suggestions regarding the new logging facilities. 

As far as I remember, I had access to both the training and dev scores during the v2 training process. Maybe I miss something, but I could not configure the training script to output both scores/ losses. Is there an easy way to achieve this?

**Comment by adrianeboyd** [comment]
Hmm, the eval intervals have changed a bit, but the available values are pretty much the same as in v2: losses based on the training steps and scores based on dev data. Can you give an example of the output that you're getting and what kind of output you're looking for?

**Reply by oroszgy** [reply]
Thanks for taking the time to reply! Indeed I am getting both losses and scores, but none of them is computed on both the training and validation sets. Basically, I am seeking a way to compare model performance measures on the train and validation data during the training process to make conclusions on its generalization capabilities.
